app/Http/Controllers/TodoController.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Support\Facades\Auth;
+use App\Http\Resources\Todo as TodoResource;
+use App\Http\Resources\TodoCollection as TodoCollectionResource;
+use App\Todo;
+use Illuminate\Http\Request;
+use DB;
+
+class TodoController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $todos = DB::table('todos')->where('users_id', Auth::id())->get();
+        return $todos;
+    }
+
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $request->validate([
+            'title' => 'required',
+            'priority' => 'required',
+        ]);
+        $todo = Todo::create($request->all() + ['users_id' => Auth::id()]);
```

Comment:
`+`  ??
`array_merge($request->validated(), [user_id => Auth...])` tako
to validated ce raditi tek kad napravis request i ubacis ga tu